api/src/main/kotlin/edu/wgu/osmt/richskill/RichSkillController.kt

```diff
@@ -219,4 +220,24 @@ class RichSkillController @Autowired constructor(
 
         return Task.processingResponse(task)
     }
+
+    @Transactional(readOnly = true)
+    @PostMapping(RoutePaths.SKILL_CSV_LIST, produces = [MediaType.APPLICATION_JSON_VALUE])
+    @ResponseBody
+    fun exportCustomList(
+        @RequestBody uuids: List<String>?,
+        @AuthenticationPrincipal user: Jwt?
+    ): HttpEntity<TaskResult> {
+        if (!appConfig.allowPublicSearching && user === null) {
+            throw GeneralApiException("Unauthorized", HttpStatus.UNAUTHORIZED)
+        }
+        if (!oAuthHelper.hasRole(appConfig.roleAdmin)) {
```

Comment:
Not limited to admins.

Reply:
removed this check, but for some reason GH is not reflecting the change here.